app/crawler/fetcher.py

```diff
@@ -0,0 +1,29 @@
+from selenium import webdriver
+import requests
+
```

Comment:
interesting to see that you write several Fetcher classes. What's the advantage of that instead of writing a fetcher class with multiple subfunctionalities?